### PR TITLE
Transfer leg support

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Supports rendering "transfer" legs correctly. They are just informational, there's no directions that should be shown

### CODE REVIEW

File: packages/itinerary-body/src/defaults/access-leg-description.tsx [hunk 1]
```diff
@@ -111,6 +113,23 @@ export default function AccessLegDescription({
...
+          description="Summarizes an transfer leg"
+          id="otpUi.AccessLegBody.transfer"
+          values={{
+            duration: intl.formatMessage(
+              {
+                id: "otpUi.ItineraryBody.common.durationShort"
+              },
+              {
+                approximatePrefix: false,
+                ...coreUtils.time.toHoursMinutesSeconds(leg.duration)
```
Comment (binh-dam-ibigroup) [inline packages/itinerary-body/src/defaults/access-leg-description.tsx:128]: Nit: I see a call to `formatDuration` above, could we use that here?

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM, see minor comments.

File: packages/itinerary-body/src/defaults/access-leg-description.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/itinerary-body/src/defaults/access-leg-description.tsx:128]: Both behave slightly differently! Also `formatDuration` is not guaranteed to be present

File: packages/itinerary-body/src/defaults/access-leg-description.tsx [hunk 2]
```diff
@@ -111,6 +113,23 @@ export default function AccessLegDescription({
             place: placeContent
           }}
         />
+      ) : isTransferLeg ? (
+        <FormattedMessage
+          defaultMessage="Transfer, wait {duration}"
+          description="Summarizes an transfer leg"
```
Comment (daniel-heppner-ibigroup) [inline packages/itinerary-body/src/defaults/access-leg-description.tsx:119]:
```suggestion
          description="Summarizes a transfer leg"
```

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Code looks good, wondering if we should really display the walk icon for transfer legs.

**Comment by miles-grant-ibigroup** [comment]
What icon are you looking for?

**Comment by miles-grant-ibigroup** [comment]
So the snapshots are making it seem like there's some weird stuff going on with TNC legs, but this seems to be a snapshot issue more than an actual issue.

**Pull request merged**